<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondpageuser</class>
 <widget class="QDialog" name="secondpageuser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/.qtcreator/thumb-1920-1398872.jpg);</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: light gray;
color: rgb(226, 255, 243);
text-decoration: underline;
font: 25pt &quot;Segoe Print&quot;</string>
   </property>
   <property name="text">
    <string>CinemaXx</string>
   </property>
  </widget>
  <widget class="QLabel" name="photolabel1">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/KIIT/Downloads/avatar.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="photolabel2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/KIIT/Downloads/nemo.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="photolabel3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/KIIT/Downloads/angry bird.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="select1">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>SELECT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SELECT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SELECT</string>
   </property>
  </widget>
  <widget class="QLabel" name="photolabel4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/KIIT/Downloads/1.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="select4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SELECT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BACK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration4">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>292</y>
     <width>74</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Duration:Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name4">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>271</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name of Movie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="language4">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>313</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration3">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>292</y>
     <width>74</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Duration:Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name3">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>271</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name of Movie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="language3">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>313</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name2">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>271</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name of Movie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration2">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>292</y>
     <width>74</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Duration:Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="language2">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>313</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QLabel" name="language1">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>313</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration1">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>292</y>
     <width>74</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Duration:Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>271</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name of Movie:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
